### ISSUE TITLE
v6.0.0 contains .NET SDK 6.0.418 instead of the latest LTS (8.0.x)

### ISSUE THREAD

**Issue by adamralph** [super-linter/super-linter#5243]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

As can be seen from [this build log](https://github.com/adamralph/bullseye/actions/runs/7812836345/job/21310694008?pr=942#step:4:370), .NET SDK 6.0.418 is present (and .NET SDK 8.0.x is not).

### Expected Behavior

I would expect .NET SDK 8.0.x to be present. Take note of https://github.com/super-linter/super-linter/pull/5047

### Super-Linter version

```markdown
v6.0.0
```

### Relevant log output

```shell
6.0.418 [/usr/share/dotnet/sdk]
  ------
  2024-02-07 09:47:23 [INFO]   Command output for CSHARP:
  ------
  The command could not be loaded, possibly because:
    * You intended to execute a .NET application:
        The application 'format' does not exist.
    * You intended to execute a .NET SDK command:
        A compatible .NET SDK was not found.
  
  Requested SDK version: 8.0.100
  global.json file: /github/workspace/global.json
  
  Installed SDKs:
  
  Install the [8.0.100] .NET SDK or update [/github/workspace/global.json] to match an installed SDK.
  
  Learn about SDK resolution:
  https://aka.ms/dotnet/sdk-not-found
```

### Steps To Reproduce

1. Specify .NET SDK 8.x in `global.json`:
   ```json
   {
     "sdk": {
       "version": "8.0.100",
       "rollForward": "latestMajor"
     }
   }
   ```
2. Try and use super-linter v6.0.0

**Comment by ferrarimarco** [comment]
Hi!

I think the issue is that we install the .NET SDK using the `channel` option, that [works as follow](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#options):

```
-Channel|--channel <CHANNEL>

Specifies the source channel for the installation. The possible values are:

STS: The most recent Standard Term Support release.
LTS: The most recent Long Term Support release.
Two-part version in A.B format, representing a specific release (for example, 3.1 or 6.0).
Three-part version in A.B.Cxx format, representing a specific SDK release (for example, 6.0.1xx or 6.0.2xx). Available since the 5.0 release.
```

Supported LTS and STS versions are available [here](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle).

As you noted, in #5047 we set it to `LTS` (it was set to `STS` before), but the behavior is the similar.

I think we should just set it to a fixed value (`8.0` in this case), instead of selecting a channel.

Also, it would make sense to have a `global.json` defined for our test environment so we can catch this issue when building the image.

**Comment by adamralph** [comment]
@ferrarimarco please see https://github.com/super-linter/super-linter/pull/5244
